setup.py

```diff
@@ -30,7 +30,7 @@
     url="https://github.com/marcoalopez/GrainSizeTools",
     license=license,
     packages=find_packages(),
-    scripts=['grain_size_tools/GrainSizeTools_script.py', 'grain_size_tools/tools.py', 'grain_size_tools/plots.py', 'grain_size_tools/piezometers.py'],
+    # scripts=['grain_size_tools/GrainSizeTools_script.py', 'grain_size_tools/tools.py', 'grain_size_tools/plots.py', 'grain_size_tools/piezometers.py'],
```

Comment:
Here, rather than installing each script separately, we the entire module will be installed as `grain_size_tools`, and each accessible under that namespace. e.g.
```
grain_size_tools.tools
grain_size_tools.plots 
... 
```